src/main/java/com/minecolonies/coremod/entity/ai/citizen/smelter/EntityAIWorkSmelter.java

```diff
@@ -89,15 +101,16 @@ public EntityAIWorkSmelter(@NotNull final JobSmelter job)
     {
         super(job);
         super.registerTargets(
-                new AITarget(SMELTER_SMELTING_ITEMS, this::smeltStuff)
+          new AITarget(SMELTER_SMELTING_ITEMS, this::smeltStuff)
         );
         worker.setSkillModifier(STRENGTH_MULTIPLIER * worker.getCitizenData().getStrength()
-                + INTELLIGENCE_MULTIPLIER * worker.getCitizenData().getIntelligence());
+                                  + INTELLIGENCE_MULTIPLIER * worker.getCitizenData().getIntelligence());
         worker.setCanPickUpLoot(true);
     }
 
     /**
      * He will smelt down armor, weapons and tools to smaller pieces here.
+     *
      * @return the next state to go to.
      */
     private AIState smeltStuff()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 22 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)

Reply:
That's okay, you can leave this.